cli/src/pkg/users.py

```diff
@@ -101,7 +111,7 @@ def addUsers(configObj):
     try:
         err = createUserFiles(userList, path + "/files")
         utils.checkError(err)
-        err = addUsersToCompose(userList, compose, server, path)
+        err = addUsersToCompose(userList, compose, server, path, configObj)
```

Comment:
passing the config object to `addUsersToCompose` function might not be desirable. Why not just pass the dictionary of resource limits?